### PR TITLE
Adding a tr-suite-master trigger file

### PR DESCRIPTION
**Pull request by emesika**
Adds a tr-suite-master trigger file in order to run OST on
tr-suite-master suite using the passed PR URL and distro

Signed-off-by: Eli Mesika emesika@rehat.com

## Please describe the change you are making

...

## Are you the owner of the code you are sending in, or do you have permission of the owner?

...

## The code will be published under the BSD 3 clause license. Have you read and understood this license?

...

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by emesika** [comment]
ci test please

**Review by janosdebugs — APPROVED** [review]
@emesika you may need to talk to @michalskrivanek to get OST access for this repo.

**Comment by michalskrivanek** [comment]
nothing special needed, the action won't work until merged. i don't think you want to merge it with the hardcoded PR though.

**Comment by janosdebugs** [comment]
Right. @emesika can you insert the PR number there?

File: .github/tr-master-suite-trigger.yaml
Comment (janosdebugs) [inline .github/tr-master-suite-trigger.yaml:18]: Please change this. :)

**Review by janosdebugs — CHANGES_REQUESTED** [review]

File: .github/tr-master-suite-trigger.yaml
Comment (emesika) [reply .github/tr-master-suite-trigger.yaml]: done

**Review by janosdebugs — APPROVED** [review]

**Pull request merged**
